<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grade_calculator</class>
 <widget class="QMainWindow" name="grade_calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>grade_calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="slider_hw1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hw2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hw3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hw4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hw5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hw6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hw7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_hw8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_hw1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_hw2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_hw3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_hw4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_hw5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_hw6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_hw7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_hw8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>500</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Pic 10B. Interemediate Programming</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odracir Razalas, SULP-SULP-EES</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="slider_mt1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_mt2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_fin">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>250</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="scA">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Schema A</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="scB">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Schema B</string>
    </property>
   </widget>
   <widget class="QLabel" name="hw1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hw 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="hw2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hw 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="hw3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hw 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="hw4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hw 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="hw5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hw 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="hw6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hw 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="hw7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hw 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="hw8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hw 8</string>
    </property>
   </widget>
   <widget class="QLabel" name="mt1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="mt2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="final_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Final</string>
    </property>
   </widget>
   <widget class="QLabel" name="course">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COURSE</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_mt1">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_mt2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>280</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_fin">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>380</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="overall_score">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>600</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="score_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>600</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OVERALL SCORE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
